WordPressAuthenticator.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
-  s.name          = "WordPressAuthenticator"
-  s.version       = "1.41.0-beta.1"
+  s.name          = 'WordPressAuthenticator'
+  s.version       = '1.41.0-beta.3'
```

Comment:
Because https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/610 added `1.41.0-beta.2`.
Will update to proper latest version + add a tag before final merge.